README.md

```diff
@@ -80,6 +80,32 @@ load module in node.js style as `require('pica')` - your project MUST
 be compiled with [browserify](https://github.com/substack/node-browserify)
 to properly use Web Workers. In other case - use `require('pica/dist/pica')`.
 
+webpack:
+
+If you use Webpack to bundle your application, you must to define a resolve
+alias into your webpack config, like this:
+
+```js
+{
+    ....
```

Comment:
1. I think it's safe to drop `...` everywhere and cut down total height. It's important to keep readme compact.
2. Also, please, use 2 spaces for indent, not 4.